src/query/fetch.c

```diff
@@ -4001,6 +4077,30 @@ fetch_peek_dbval (THREAD_ENTRY * thread_p, REGU_VARIABLE * regu_var, VAL_DESCR *
 
 	  switch (funcp->ftype)
 	    {
+	    case F_JSON_ARRAY:
+	    case F_JSON_OBJECT:
+	    case F_JSON_INSERT:
+	    case F_JSON_REMOVE:
+	    case F_JSON_MERGE:
```

Comment:
please also confirm the functions.

Reply:
confirmed for json_object, json_remove and json_merge.

json_insert is a little tricky because MySQL's path language differentiates between array indices and object names, while rapidjson's path language and the one we use doesn't.

the following difference appears:

MySQL:

select json_insert (json_object('a', json_object('b', 'c')), '$.a[7]', 'c');
+----------------------------------------------------------------------+
| json_insert (json_object('a', json_object('b', 'c')), '$.a[7]', 'c') |
+----------------------------------------------------------------------+
| {"a": [{"b": "c"}, "c"]}                                             |
+----------------------------------------------------------------------+

CUBRID:

select json_insert (json_object('a', json_object('b', 'c')), '/a/7', '"c"');
  json_insert(json_object('a', json_object('b', 'c')), '/a/7', '"c"')
======================
  {"a":{"b":"c","7":"c"}}

I suggest we leave this as is for now because it is a very marginal case. In the future we might switch to the path used in MySQL or restrict the path to refer only to array indices when it contains only a number.